assets/src/scripts/web-services/groundwater-levels.js

```diff
@@ -2,6 +2,19 @@
 import {get} from 'ui/ajax';
 import config from 'ui/config';
 
+
+/**
+ *  Formats a URL for the purpose of downloading groundwater levels data in RDB format
+ * @param  {String}  siteno - A code that uniquely identifies a monitoring location
+ * @param  {String}  parameterCode - USGS five digit parameter code
+ * @param {String} startTimeDT - ISO 8601 time
+ * @param {String} endDT- ISO 8601 time
+ * @return {String} The URL used to contact waterservices.usgs.gov
+ */
+export const getServiceURLSGroundwater = function({siteno, parameterCode, startDT, endDT}) {
```

Comment:
I think you should use this in the fetch function too. Which means passing in period but that can be defaulted to null. This makes everything more DRY